### PR TITLE
Integration of latest Akamai Library

### PR DESCRIPTION
**Pull request by hardikhexa**
Integration of latest Akamai Library

### CODE REVIEW

File: components/tasks/AkamaiAnalyticsTask.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+<script type="text/brightscript" >
+<![CDATA[
+
+Function init()
+  m.top.functionName = "loadAnalytics"
+end Function
+
+Function loadAnalytics()
+  print "hello world"
```
Comment (lucianopanaro) [inline components/tasks/AkamaiAnalyticsTask.xml:18]: I'm guessing we don't need this, right?

File: source/main.brs [hunk 2]
```diff
@@ -896,23 +889,10 @@ sub playVideo(screen as Object, auth As Object, adsEnabled = false, content = in
...
-      title: content.title,
-      deviceType: playerInfo.analytics.device,
-      playerId: playerInfo.analytics.playerId,
-      contentLength: content.length,
-      consumerId: consumer_id
-    }
-    print "Custom Dimensions: "; cd
-    m.AKaMAAnalyticsPlugin.pluginMain({configXML: playerInfo.analytics.beacon, customDimensions:cd})
+
   end if
```
Comment (lucianopanaro) [inline source/main.brs:896]: Is this block needed now? It seems it was moved down to line 996

Comment (hardikhexa) [reply source/main.brs:896]: This already showing deleted lines so block already moved.
Not sure which block you are referring?

File: components/tasks/AkamaiAnalyticsTask.xml (hunk 1 above)
Comment (hardikhexa) [reply components/tasks/AkamaiAnalyticsTask.xml:18]: This is main task function and its needed.
Do you mean for "Hello World" message ? 👍 ?

File: source/main.brs (hunk 2 above)
Comment (lucianopanaro) [reply source/main.brs:896]: I'm referring to lines 892 and 893 -- it seems that print statement could be removed or moved to the block in line 996?

```
if(playerInfo.analytics.beacon <> invalid AND playerInfo.analytics.beacon <> "")
    print "PlayerInfo.analytics: "; playerInfo.analytics
```

File: components/tasks/AkamaiAnalyticsTask.xml (hunk 1 above)
Comment (lucianopanaro) [reply components/tasks/AkamaiAnalyticsTask.xml:18]: yeah the print "hello world"

Comment (hardikhexa) [reply components/tasks/AkamaiAnalyticsTask.xml:18]: But I agree to remove this.

File: source/main.brs (hunk 2 above)
Comment (hardikhexa) [reply source/main.brs:896]: sure that can be done. actually there are lots of debug and logs into code, which can be done cleanup.
But I agree to remove this.

**Pull request merged**